### PR TITLE
Fix logging port

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description
- https://jira.mesosphere.com/browse/DOCS-1683
- Change port from `8080` to `80`
- Related to https://github.com/mesosphere/dcos-docs-enterprise/pull/1035

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by adyatlov** [comment]
LGTM

File: 1.9/administration/logging/index.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ menu_order: 3.4
 
 DC/OS cluster nodes generate logs that contain diagnostic and status information for DC/OS core components and DC/OS services.
 
+The logging component provides an HTTP API (`/system/v1/logs/`) which exposes the Mesos system logs.
```
Comment (darkonie) [inline 1.9/administration/logging/index.md]: those are not mesos system logs, journald logs?
cc @malnick

Comment (darkonie) [inline 1.9/administration/logging/index.md]: or probably just `system logs`

**Review by initcore0 — COMMENTED** [review]
looks good to me, just fix the mesos system logs

**Review by initcore0 — APPROVED** [review]

**Pull request merged**